package.json

```diff
@@ -15,9 +15,13 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test-no-watch": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
     "lint": "eslint './src/**/*.js'",
-    "lint-fix": "eslint './src/**/*.js' --fix"
+    "lint-fix": "eslint './src/**/*.js' --fix",
```

Comment:
This is actually my fault, but we should update the `lint` and `lint-fix` command to check `.jsx` files too.

Would you mind making this change in this PR?

Reply:
Will do.

Reply:
Did do.